docs/scripts/generate-sitemap-robotstxt.ts

```diff
@@ -107,7 +107,7 @@ User-agent: *
 Allow: /
```

Comment:
Please also make the User-agent: * blocked for dev:
```
User-agent: *
${isProd ? `Allow: /` : `Disallow: /`}
```

See:
https://www.bing.com/webmasters/help/?topicid=cb7c31ec